config/locales/jobs.yml

```diff
@@ -21,8 +21,8 @@ en:
       how_to_accept_job_applications_guide:
         description_desktop: Find out the simplest way to get candidates to apply for your jobs.
```

Comment:
I feel like we're just repeating ourselves with the heading, description AND link text - could we get rid of these descriptions and just have the heading and link text underneath?

Reply:
Done!